lib/std/zig/Ast/Render.zig

```diff
@@ -783,7 +783,51 @@ fn renderExpression(r: *Render, node: Ast.Node.Index, space: Space) Error!void {
         => {
             var buf: [2]Ast.Node.Index = undefined;
             const params = tree.builtinCallParams(&buf, node).?;
-            return renderBuiltinCall(r, tree.nodeMainToken(node), params, space);
+            var builtin_token = tree.nodeMainToken(node);
+
+            if (params.len == 1) canonicalize: {
```

Comment:
I don't actually know if other people agree with me here, but when I have a labeled block, I prefer to move all conditions into the block to reduce the amount of control flow constructs in use:
```suggestion
            canonicalize: {
                if (params.len != 1) break :canonicalize;
```